### DISCUSSION TITLE
aria2下载

### DISCUSSION THREAD

**Discussion by qyxk** [AlistGo/alist#491] [Ideas]
能不能添加aria2下载呢，这样可以直接下载到nas上

**Comment by Windman1320** [comment]
dev分支已加入aria2下载支持，详情见 https://github.com/alist-org/alist-web/pull/42

**Comment by anwen-anyi** [comment]
> 能不能添加aria2下载呢，这样可以直接下载到nas上

@Windman1320 大佬已经搞定了,下个正式版应该会上线吧,要是自己想提前试试的话可以自己编译或者体验一下我编译好的Windows版本的先试验一下,

支持`Motrix`调用,端口号改一下即可,
「Ar2Alist尝鲜.exe」https://www.aliyundrive.com/s/Xy2o95fYD5K

启动后记得把静态资源的路径改成本地,不然会无法使用静态资源 **`/data/config.json`**
 -   "assets": "https://npm.elemecdn.com/alist-web@$version/dist",
 -   "assets": "/",  --`改成这样`
